readthedocs/core/utils/general.py

```diff
@@ -23,3 +31,17 @@ def wipe_version_via_slugs(version_slug, project_slug):
     ]
     for del_dir in del_dirs:
         broadcast(type='build', task=remove_dirs, args=[(del_dir,)])
+
+    _clear_html_files_from_cloud_storage(version)
+
+
+def _clear_html_files_from_cloud_storage(version):
+    """Removes html files from cloud storage for a given version of a project."""
+
+    storage = get_storage_class(settings.RTD_BUILD_MEDIA_STORAGE)()
```

Comment:
In other parts of the code we check for `if settings.RTD_BUILD_MEDIA_STORAGE:` but the setting is defined on our base settings. /cc @davidfischer

Reply:
Those `if` checks can now be removed, I think. Media storage is now required (although there's a PR to merge on the corporate code for this).

Reply:
I created a PR for it: https://github.com/readthedocs/readthedocs.org/pull/6191